src/API.CRUD/Extensions/JsonExceptionPageExtensions.cs

```diff
@@ -0,0 +1,146 @@
+using System;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using FluentValidation;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Cors.Infrastructure;
+using Microsoft.AspNetCore.Diagnostics;
+using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json;
+
+namespace API.CRUD.Extensions
+{
+    public static class JsonExceptionPageExtensions

```

Comment:
You'll have to show me what the methods/helpers in this class does. Will be easier to understand.

Reply:
Happy with this. Just need to tweak class to not show stack trace on prod, make sure multiple errors are supported (e.g array instead of single prop), and test inner exception is shown etc

Reply:
🔧